src/bauhaus/bauhaus.c

```diff
@@ -2595,7 +2595,7 @@ static void _slider_add_step(GtkWidget *widget, float delta, guint state, gboole
   }
   else if(!strcmp(d->format,"°") && (d->max - d->min) * d->factor == 360.0f
           && fabsf(value + delta)/(d->max - d->min) < 2)
```

Comment:
It might be worthwhile allowing a bit of fuzz on the comparison to 360, i.e.
`fabsf((d->max - d->min) * d->factor - 360.0f) < 0.0001f`
since the result after multiplying the difference may not be exact.

Reply:
This is meant to deal with 0-360, -180 - +180 and 0-1 * 360, so not scaled diffs between random close numbers. Seems to work fine on my Intel, but if there's a not-purely-theoretical chance it behaves differently on other architectures, it would be worth sacrificing some readability. Or even spell out those options. What do you think?

Reply:
I brought up the issue because one of my recent speedups was adding the fuzz factor so that the default neutral setting would actually cause the processing code to be skipped as intended - after a double-click to reset the slider to 1.0, the actual value read from three different sliders was 0.9999999 or 1.0000001.

Reply:
Can you provide a link to the change please? Not sure if you are saying you introduced this small diff on purpose or found that it unintentionally appeared in bauhaus. If first, it "shouldn't" impact the min/max values, if the latter, then, well...

Reply:
I introduced it on purpose in src/iop/colorbalance.c.
e.g. `const int run_saturation = fabsf(saturation - 1.0f) > 1e-6;`
instead of `const int run_saturation = (saturation != 1.0f);`

Reply:
Interesting...

Those are _slightly_ more complex cases in that they have
`dt_bauhaus_slider_set_soft_range(..., 0.5f, 1.5f);`, scaling factor of 100 (-100) and in one case an offset. There are even more fun ones where a whole non-linear transformation function is applied backwards and forwards so I would be more surprised if that _didn't_ introduce rounding errors.

I'll introduce a fuzz out of abundance of caution. Thanks!